src/twisted/plugins/cred_unix.py

```diff
@@ -43,7 +43,10 @@ def verifyCryptedPassword(crypted, pw):
         pw = pw.decode('utf-8')
     if not isinstance(crypted, StringType):
         crypted = crypted.decode('utf-8')
-    return crypt.crypt(pw, crypted) == crypted
+    try:
+        return crypt.crypt(pw, crypted) == crypted
+    except OSError:
+        return False
```

Comment:
Maybe conch should… just be importing this function, given that we've implemented the exact same thing in both places here?

Reply:
Will do.

Reply:
@mgorny fixed this by adding `from twisted.plugins.cred_unix import verifyCryptedPassword` in `src/twisted/conch/checkers.py`